packages/ketcher-core/src/domain/helpers/attachmentPointCalculations.ts

```diff
@@ -130,16 +130,22 @@ export function getSearchFunction(
       x: Math.round(zoomedCoordinateOfSecondPoint.x) + canvasOffset.x,
       y: Math.round(zoomedCoordinateOfSecondPoint.y) + canvasOffset.y,
     };
-    const newPoint = document.elementFromPoint(
-      newPointCoord.x,
-      newPointCoord.y,
-    ) as Element;
-
-    let newAngle;
-    if (newPoint === monomer.renderer?.bodyElement?.node()) {
-      newAngle = initialAngle;
-    } else {
-      newAngle = initialAngle - 180;
+    let newAngle: number = initialAngle;
+
+    if (monomer.renderer?.bodyElement) {
```

Comment:
I propose to remove this line since we already used Optional chaining (?.) for it